src/lib/components/molecules/table/table.stories.jsx

```diff
@@ -36,6 +36,7 @@ export const Default = {
         lastName: 'Smith',
       },
     ],
+    hideHeader: false,
```

Comment:
Sorry to be pedantic, but we can now remove this prop from the stories

Reply:
oh okay - sure - well we should keep it in one at least to document the option. I'll keep it in the party example